pkg/controller/podgc/gc_controller_test.go

```diff
@@ -657,6 +661,133 @@ func TestGCTerminating(t *testing.T) {
 	testDeletingPodsMetrics(t, 7)
 }
 
+func TestGCDeleteAllPodsBoundToNode(t *testing.T) {
+	testNode := &v1.Node{
```

Comment:
nit: The node doesn't seem needed for orphan pods

Reply:
Removed